Platforms/QemuSbsaPkg/Plugins/QemuRunner/QemuRunner.py

```diff
@@ -159,4 +162,11 @@ def Runner(env):
             # Tested same FDs on QEMU 6 and 7, not observing the same.
             ret = 0
 
+        if os.name == 'nt' and qemu_version[0] >= '8':
+            # Restore the console mode for Windows on QEMU v8+.
+            std_handle.SetConsoleMode(console_mode)
+        elif os.name != 'nt':

```

Comment:
Does this need the same >= '8' condition?

Reply:
Not really, the Linux versions have been giving trouble to the runner console for more than a year (if not 2 years). We definitely want to restore the `stty` state when the return code is incorrect, but I agree with Oliver that a clean up seems to be needed regardless as QEMU kept polluting the returned console here and there. Let me know if you have any concerns.